/**:
  ros__parameters:
    portname: "/dev/logic_distro_xiao"
    baudrate: 115200
    mcu_ip_addr: "192.168.9.5"
    mcu_port: 9011
    this_port: 9011
    timeout_ms: 5000
